src/twisted/conch/scripts/cftp.py

```diff
@@ -54,12 +62,14 @@ def parseArgs(self, host, localPath=None):
         self['host'] = host
         self['localPath'] = localPath
 
+
+
 def run():
-#    import hotshot
-#    prof = hotshot.Profile('cftp.prof')
-#    prof.start()
+    # import hotshot
+    # prof = hotshot.Profile('cftp.prof')
+    # prof.start()
```

Comment:
Looks like hotshot was removed from cpython in 2007: https://github.com/python/cpython/commit/0e474a801ac6e01a24b900183ef101962148f317

so this can be removed